src/Sentry/Internal/Json.cs

```diff
@@ -15,5 +16,12 @@ public static JsonElement Parse(string json)
             using var jsonDocument = JsonDocument.Parse(json);
             return jsonDocument.RootElement.Clone();
         }
+
+        public static JsonElement Load(string filePath)
+        {
+            using var file = File.OpenRead(filePath);
+            using var jsonDocument = JsonDocument.Parse(file);
+            return jsonDocument.RootElement.Clone();
```

Comment:
`jsonDocument.RootElement` is a JSON element backed by a streaming/lazy `JsonDocument`. What `Clone()` does is buffer it all in memory so that the original stream can be safely disposed.